misc/life_mngr/Makefile

```diff
@@ -0,0 +1,49 @@
+T := $(CURDIR)
+OUT_DIR ?= $(shell mkdir -p $(T)/build;cd $(T)/build;pwd)
+CC ?= gcc
+
+LIFEMNGR_CFLAGS := -g -O0 -std=gnu11
+LIFEMNGR_CFLAGS += -D_GNU_SOURCE
+LIFEMNGR_CFLAGS += -DNO_OPENSSL
+LIFEMNGR_CFLAGS += -m64
+LIFEMNGR_CFLAGS += -Wall -ffunction-sections
+LIFEMNGR_CFLAGS += -Werror
+LIFEMNGR_CFLAGS += -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2
```

Comment:
Is it reasonable to specify both O0 and O2 at the same time?

Reply:
This Makefile refers to other tool's Makefile.
In order to keep them uniformity.

Reply:
But it is not right to do so, for this patch,  just keep O2? need other patch to correct the other Makefile

Reply:
OK. I will correct it.